package.json

```diff
@@ -31,8 +31,10 @@
         "uuid": "^8.3.2"
     },
     "devDependencies": {
+        "@nomiclabs/hardhat-ethers": "^2.2.3",
```

Comment:
now that tests were removed this shouldn't be needed?

Reply:
removed